src/Tasks/Microsoft.Managed.After.targets

```diff
@@ -41,10 +41,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <_MainReferenceTargetForBuild Condition="'$(BuildProjectReferences)' == '' or '$(BuildProjectReferences)' == 'true'">.projectReferenceTargetsOrDefaultTargets</_MainReferenceTargetForBuild>
     <_MainReferenceTargetForBuild Condition="'$(_MainReferenceTargetForBuild)' == ''">GetTargetPath</_MainReferenceTargetForBuild>
 
-    <ProjectReferenceTargetsForBuildInOuterBuild>GetTargetFrameworks;$(ProjectReferenceTargetsForBuildInOuterBuild)</ProjectReferenceTargetsForBuildInOuterBuild>
+    <ProjectReferenceTargetsForBuildInOuterBuild>$(ProjectReferenceTargetsForBuildInOuterBuild)</ProjectReferenceTargetsForBuildInOuterBuild>
```

Comment:
This is now a nop. Same with `ProjectReferenceTargetsForCleanInOuterBuild` below

Reply:
The top of the file has
```
 Properties for extension of ProjectReferenceTargets.
      Append any current value which may have been provided in a Directory.Build.props since the intent was likely to append, not prepend.
      ```
I thought the current value referenced here would be any preexisting property definitions in Directory.Build.props?

Reply:
So this is intended to preserve existing values, but effectively you're changing from:

```cs
string foo = "bar;" + foo;
```

to:

```cs
string foo = foo;
```

So the updated line doesn't actually do anything now. You're assigning it to its current value.